Mars-SeleniumAutomation/Process/LanguageProcess.cs

```diff
@@ -0,0 +1,182 @@
+﻿using AventStack.ExtentReports;
+using Mars_SeleniumAutomation.Pages.Components.ProfilePageOverview;
+using Mars_SeleniumAutomation.TestModel;
+using Mars_SeleniumAutomation.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using RazorEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mars_SeleniumAutomation.Process
+{
+    public class LanguageProcess : Base
+    {
+        AddUpdateLanguageComponent addUpdateLanguageComponentObj;
+        Jsonhelper jsonhelperObj;
+        Base baseObj;
+        ExtentTest testreport;
+        private ExtentReports extent;
+#pragma warning disable
+        public LanguageProcess()
+        {
+            addUpdateLanguageComponentObj = new AddUpdateLanguageComponent();
+            baseObj = new Base();
+            jsonhelperObj = new Jsonhelper();
+        }
+        public void addedLanguageDetails()
+        {
+            baseObj.InitializeExtentReports();
+            string testName = "LanguageAddData";
+            baseObj.SetupTest(testName);
+            LanguageTestModel model = new LanguageTestModel();
+            List<LanguageTestModel> LanguageAddData = Jsonhelper.ReadTestDataFromJson<LanguageTestModel>("D:\\IC Course\\AdvancedNUnit\\Mars-AdvancedTaskNUnit\\Mars-SeleniumAutomation\\JsonData\\LanguageAddData.json");
+            foreach (var data in LanguageAddData)
+            {
+                string language = data.language;
+                Console.WriteLine(language);
```

Comment:
remove console writeline every where it is bad practices to include this in code